### DISCUSSION TITLE
image resizing

### DISCUSSION THREAD

**Discussion by Fkaule** [ObsidianPublisher/obsidian-github-publisher#253] [Q&A — Plugin] [answered]
Hello,

I want to resize an .png image like so for the maximum with of 300, which works in obsidian:

```
![[image.png|300]]
````

Is there another way to do this? I know that with the standard markdown syntax the resizing is possible with the `{width="300"} ` attribute.

It would be awesome if ObsidianPublisher could translate this: 

```
![[filename with spaces.png|300]]
````

to this:

```
![filename with spaces](<image path with spaces/filename with spaces.png>){width="300"} 
````

Thank you very much !! :)

**Comment by Mara-Li** [comment]
Do you use my template for Mkdocs ? Because native Obsidian format works (normally).

Otherwise, you could use the regex replace for text and use it in arrow down order (after all other transformation). 

Test here : https://regex101.com/r/JqHvlz/1

**Reply by Fkaule** [reply]
I used your template!

The image is embedded just fine, but the resizing in the "Obsidian way" e.g. ![[image.png|300]] for 300px  doesn't work properly. 

Where can I embed the regex command?

**Reply by Mara-Li** [reply]
Pretty strange as it works on my side... 
You use wikilinks syntax in mkdocs? Or transformation into Markdown links?

**Reply by Fkaule** [reply]
Where can I set up the wikilinks syntax in mkdocs? I use Obisdian and your "Obsidian Publisher" within github. 

I created a small example: https://fkaule.github.io/mini_example/ .

I used this syntax in Obsidian `![[BeispielA_Aufgabenstellung.png|200]]` and it renders fine with the right size within Obsidian, but for the github page it does not. Although it's working with `![Image title](BeispielA_Aufgabenstellung.png){width="300"}` for the github page but than it's not correctly rendered by Obsidian :-(

**Reply by Mara-Li** [reply]
Can I get the github repository connected to the minimal exemple and also your configuration ? 

by the way, please read the documentation. You will found a lot of useful option in the settings.

**Reply by Mara-Li** [reply, marked as answer]
**update** : It was a bug, caused by a code duplication in the javascript. I updated them, so you need to wait to the purge cache.

**Reply by Fkaule** [reply]
Thank you very much for you help! But it still doesn't work :-( What do I need to do?

I updated your Obsidian Plugin. The `Get latest release from template` action doesn't work properly, maybe that's why it's not working? I get the error: `Input 'token' not supplied. Unable to continue.` 

I created `GH_TOKEN` (with `repo` and `workflow` permission) and `AUTHOR_EMAIL` and `AUTHOR_NAME` (I use a team repo and used my name here) repo secrets. Is there anything else?

**Reply by Mara-Li** [reply]
You just need to clean your cache. It's already up on my side.

**Reply by Mara-Li** [reply]
(it's an hosted JS so you really nothing to do with that)

**Reply by Fkaule** [reply]
Oh thank you very much, you are my hero 👍
